index.Rmd

```diff
@@ -79,21 +99,76 @@ Tables and figures (maps and other graphics) are carefully planned to convey the
 
 Show tables, plots, etc. and describe them.
 
-```{r, fig.width=6, fig.height=3, fig.cap="Map of completely random data"}
-m <- leaflet(data) %>% 
-  addTiles() %>% 
-  addCircleMarkers(~x, ~y, radius = ~size,color = ~as.factor(category)) %>% 
-  addPopups(~x[2], ~y[2], "Random popup")
-m  # a map with the default OSM tile layer
+```{r}
+# 2017 - 2018 Single Family Housing Price Histogram
+Plot_2017 <- ggplot(data = Buffalo_17, mapping = aes(x = TOTAL.VALUE)) + 
+  geom_histogram() + xlab("Total Property Value($)") + ylab("Count") +
+  scale_fill_manual(values="lightblue") + theme_few() +
+  labs(x="Total Value($)", y="Count", title="Distribution of Buffalo Home Prices",
+       subtitle="Single Family Property Prices (2017 - 2018)", 
+       caption="Source: Buffalo Open Data") + scale_x_continuous() + scale_y_continuous()
+plot(Plot_2017)
 ```
 
 
 ```{r}
-data %>% 
-  ggplot(aes(x=x,y=y,col=category))+
-  geom_point()
+# 2019 - 2020 Single Family Housing Price Histogram
+Plot_2019 <- ggplot(data = Buffalo_20, mapping = aes(x = TOTAL.VALUE)) + 
+  geom_histogram() + xlab("Total Property Value($)") + ylab("Count") +
+  scale_fill_manual(values="lightblue") + theme_few() +
+  labs(x="Total Value($)", y="Count", title="Distribution of Buffalo Home Prices",
+       subtitle="Single Family Property Prices (2019 - 2020)", 
+       caption="Source: Buffalo Open Data") + scale_x_continuous() + scale_y_continuous()
+plot(Plot_2019)
+```
+
+```{r}
+#Data is transformed using Natural Log for Map Visualization
+Buffalo_17$logprice <- log(Buffalo_17$TOTAL.VALUE)
```

Comment:
I think it's generally better to log transform the color scales rather than the values - see below